src/System.Net.Security/src/System/Net/Windows/SSPIAuthType.cs

```diff
@@ -0,0 +1,213 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Runtime.InteropServices;
+using System.Globalization;
+using System.Net.Security;
+using System.Security.Cryptography;
+using System.Security.Cryptography.X509Certificates;
+using System.ComponentModel;
+using System.Security.Principal;
+using System.Security.Authentication.ExtendedProtection;
+using System.Security;
+using System.Collections;
+using System.Diagnostics;
+using Microsoft.Win32.SafeHandles;
+using System.Security.Authentication;
+
+namespace System.Net
+{ 
+
+    // For Authentication (Kerberos, NTLM, Negotiate and WDigest):
+    internal class SSPIAuthType: SSPIInterfaceNego
```

Comment:
This code has changed from the original, oddly enough, without any apparent reason.
I diffed by extracting to another file and diffing that way.